.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -106,8 +108,27 @@ jobs:
         run: bash tests/integration/kubernetes/gha-run.sh run-tests
 
       - name: Report tests
+        id: report-tests
         if: always()
-        run: bash tests/integration/kubernetes/gha-run.sh report-tests
+        run: |
+          OUTPUT=$(bash tests/integration/kubernetes/gha-run.sh report-tests 2>&1)
+          echo "$OUTPUT"
```

Comment:
nit: Can you add braces around the variables as it's our preferred style in kata. Thanks